### DISCUSSION TITLE
How do I deploy in Netlify.

### DISCUSSION THREAD

**Discussion by snmbala** [jpanther/congo#356] [Q&A] [answered]
Without a config file in route folder and the theme file, netlify is not able to build. But it works perfectly fine in local machine. Can somebody help me how do I deploy it in Netlify.

**Comment by jpanther** [comment, marked as answer]
Hi @snmbala, are you following the [Netlify deployment docs](https://jpanther.github.io/congo/docs/hosting-deployment/#netlify) when you're trying to get this working? It's difficult to troubleshoot without further information.

**Reply by snmbala** [reply]
It works perfectly fine in the local hugo build comment. But when I try to deploy in netlify I'm getting the following error. 
<img width="500" alt="Screenshot 2022-11-09 at 10 24 33 AM" src="https://user-images.githubusercontent.com/3702953/200742114-51841072-7ad0-4389-9600-67d1e42a8b40.png">

**Reply by jpanther** [reply]
That looks like you're trying to use Jekyll but Congo is a theme for Hugo. You will need to follow the directions I linked to above to get your site to build correctly with Netlify.

**Reply by snmbala** [reply]
Yes. Even I was surprised how I'm getting such a error. But anyhow, I was able to fix it now. Instead of linking it as a submodule, I have copied the theme under theme folder. It work fine that way. And thanks for such a lovely theme. Lot of love ♡ ♡ ♡  ♡ ♡
